_posts/2023-03-20-semantic-science-benchmarks.md

```diff
@@ -0,0 +1,353 @@
+
+In this blog, we have included several experiments that can help build intuition about how and when to combine BM25 with neural retrievers. It is important to remember that because every dataset is different, so there is a chance that the configurations used here are not optimal for your dataset. Nevertheless, we believe that there are some **global conclusions** that apply to most datasets: 
+
+1. Neural retrievers with BM25 work better than neural retrievers or BM25 alone.
+2. Neural retrievers with BM25 deliver the same (or better) results as cross-encoders at a fraction of the cost and latency.
+3. If a dataset is heavy on keyword usage, BM25 works much better than neural retrievers. An example of such dataset is factory part numbers.
+4. If a dataset is heavy on natural language, neural retrievers work much better than BM25. An example of such dataset is a community forum.
+5. For datasets that contain both natural language and keywords, a combination of BM25 and neural retrievers works better. An example of such dataset is a clothing website that describes the product in natural language (product description) and numbers (product length, size, or weight).
+6. The optimal combination method depends on the dataset, however in general we have found harmonic mean to perform best for pretrained models and arithmetic mean to perform best for fine-tuned models.
+
+
+
+## Appendix: Details of test datasets
+
+In this appendix, we provide further details on the test datasets used for benchmarking. Our primary data sources were the BEIR challenge and the Amazon ESCI datasets. 
+
+
+### The BEIR dataset
+
+The BEIR challenge dataset was introduced in [a 2021 paper presented at NeurIPS](https://arxiv.org/abs/2104.08663)[.](https://arxiv.org/abs/2104.08663) It consists of 18 test datasets that cover several domains, from personal advice on Yahoo Answers to Stack Exchange questions about quantum physics. The datasets also come in different evaluation formats, for example, fact checking, question answering, and news retrieval. We used nine of the 19 datasets from the BEIR challenge. We did not use the MS Marco and NQ datasets because the query generator was trained on this data and thus it is not zero-shot. We did not benchmark on datasets that are not publicly available without registering (BioASQ, Signal-1M, Trec-News and Robust04). In the future, we plan to benchmark large (more than 5M documents each) datasets, such as fever, climate-fever and HotpotQA.
```

Comment:
Confirm that the period terminating the first sentence is intentionally linked separately.